tantivy/tantivy.pyi

```diff
@@ -197,6 +197,10 @@ class Query:
     def all_query() -> Query:
         pass
 
+    @staticmethod
+    def disjunction_max_query(subqueries: list[Query], tie_breaker: Optional[float]) -> Query:
```

Comment:
```suggestion
    def disjunction_max_query(subqueries: list[Query], tie_breaker: Optional[float] = None) -> Query:
```

[nit] According to PyO3 doc, `tie_breaker` would be automatically set to `None` (not tested.) If so, specifying that might be useful for users to save typing a bit.

> As a convenience, functions without a #[pyo3(signature = (...))] option will treat trailing Option<T> arguments as having a default of None.

https://pyo3.rs/v0.21.0/function/signature#trailing-optional-arguments

Reply:
It does seem to set `None` automatically. Currently both of the following calls work:
```
query = Query.disjunction_max_query([query1, query2])
query = Query.disjunction_max_query([query1, query2], tie_breaker=0.5)
```
In the latest commit, I added a default value of `None` to the Python signature. Does this address the comment or did I miss the point?

Reply:
Thank you for including my suggestion in the commit!